packages/gpl-core/src/connection.ts

```diff
@@ -62,7 +62,7 @@ export class Connection {
         cl_pubkey
       }
     `;
-    const result = await this.sdk.gqlClient.request(query);
+    const result: any = await this.sdk.gqlClient.request(query);
```

Comment:
Why is result explicitly set to any? @vjgpt is working on a ticket that adds type to GQL responses, if that's the motivation.